crates/bevy_sprite/src/mesh2d/mesh.rs

```diff
@@ -232,12 +232,7 @@ impl FromWorld for Mesh2dPipeline {
                 &image.data,
                 ImageDataLayout {
                     offset: 0,
-                    bytes_per_row: Some(
-                        std::num::NonZeroU32::new(
-                            image.texture_descriptor.size.width * format_size as u32,
-                        )
-                        .unwrap(),
-                    ),
+                    bytes_per_row: Some(image.texture_descriptor.size.width * format_size as u32),
```

Comment:
I'm not sure why this change is in this PR... this has already been done on `main`, and either your branch is not up to date with `main` and you shouldn't need it anyway as it's from the `wgpu` update, or it should already be done

Reply:
I haven't a clue why this happened too. Should be fixed now...